### PR TITLE
Project Requirements are Fulfilled

### PR DESCRIPTION
**Pull request by grauJavier**
### Completed Project Requirements

The project requirements have been successfully fulfilled. The following tasks have been accomplished:

**API Selection:**
- An appropriate API has been chosen for the development of the web application. The selected API allows retrieval of numeric values based on a parameter. The choice aligns with the project's objectives and requirements.

**Design:**
- The web application's design adheres to the provided [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). The chosen color scheme, typography, and layout elements have been implemented to create a visually appealing user interface.
- Credit has been appropriately attributed to the original designer, Nelson Sakwa, as required by the [Creative Commons license](https://creativecommons.org/licenses/by-nc/4.0/). This credit is prominently featured in the project's README.

**Interactions:**
- The home page of the web application functions as intended:
  - Upon loading, the page displays a list of categories that can be filtered by various parameters, such as category name.
  - Numeric values associated with each category, sourced from the API, are displayed alongside the category name.
  - Clicking or tapping on a category item navigates the user to the corresponding details page.
- The details page retrieves data from the API to provide comprehensive information about the selected category.
- The "Back" button (<) on the details page successfully returns the user to the home page, maintaining seamless navigation.

**Testing:**
- Unit tests have been created for pure functions within the Redux codebase.
- Integration tests have been developed using the React Testing Library. Mocking techniques have been employed to prevent actual API requests and Redux Store interactions during testing.

**Technical Implementation:**
- The project is a fully functional single-page application (SPA) developed using React and Redux.
- Data obtained from the API is efficiently stored within the Redux store, enabling streamlined data management and retrieval.
- A Filter stateless component has been implemented to facilitate data filtering from the API.
- Each page within the SPA, including the main page and individual item pages, has been assigned a unique route for effective routing.
- The web application has been successfully deployed and is accessible online through [Render](https://render.com), [Netlify](https://www.netlify.com/), or [GitHub Pages](https://pages.github.com/). A direct link to the deployed version of the application has been included in the project's README.

### Project Documentation:

Upon completion of the development phase, a video presentation has been recorded to showcase the project's features. The video, with a maximum duration of 5 minutes, encompasses the following content:

- A comprehensive description of the project's purpose and functionality.
- A live demonstration highlighting key features and interactions within the web application.
- A spotlight on noteworthy code snippets or accomplishments that contribute to the project's success.

For the video recording, a tool like [Loom](https://www.loom.com/) has been utilized to provide a synchronized display of the computer screen and the presenter's face. This approach ensures clear and engaging communication of the project's achievements and innovations.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,322 @@
...
+---
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (AlphaNtihinduka) [inline README.md]: - [x] Kindly ensure that the `MIT` is not broken. The MIT license gives users :person_curly_hair:  express permission to reuse code for any purpose ☺️.

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @grauJavier 🙋‍♂️
Good job working on your Air Metrics application. You have done the design to the best expectation and incredibly well. 👏  

You still need to implement a few on the project but you are almost there!!! 🚶 : 

TO HIGHLIGHT:
- Correct use of routes ✔️ 
- Proper use of async and await. ✔️ 
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

File: README.md (hunk 1 above)
Comment (grauJavier) [reply README.md]: @AlphaNtihinduka Issue addressed. I'll be waiting for your blessing! :)

**Review by AlphaNtihinduka — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
## Hi @grauJavier 🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 

approved-accepted

Congratulations! 🎉 
TO HIGHLIGHT:

- Correct use of routes ✔️ 
- Proper use of async and await. ✔️ 
- Well-structured files ✅

Now you can merge this branch and move on :1st_place_medal: 


## Optional suggestions
- Kindly refer to the inline optional suggestions made. Ensure to apply them to have a professional project that suits recruiters' needs.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:

**Comment by PabloBona** [comment]
Vamo !

**Pull request merged**